lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -947,7 +948,7 @@ void TypeAnalysis::visitOperation(Operation *op,
   if (auto sumDimIntList = dyn_cast<AtenSumDimIntListOp>(op)) {
     Type defaultDtype = operands[0]->getValue().dtype;
     // If the input dtype is bool, the result type should be i64.
```

Comment:
I don't think this is correct. You need to bail out if you don't know the defaultDtype.